topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.md

```diff
@@ -0,0 +1,220 @@
+find them, however, one generally needs to use a brute force approach to explore
+all the possible positions in which the muon may end up. {AIRSS} is one such
+method {% cite Liborio2018 %}.
+
+In this workflow, we use the {UEP} Method: a method that requires only {DFT}
+calculation, which is used to compute the electronic density of the host
+material {% cite Sturniolo2020 %}. This, in turn, is used to calculate the
+minima of the host material’s electrostatic potential and to estimate the
+stopping site of the diamagnetic muon, relying on the approximation that the
+muon’s presence does not significantly affect its surroundings.
+
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Preparation steps
+First, we need to download some simple example data and a workflow for finding
+possible stopping sites. For this, we use a CASTEP `.cell` file containing two
+Silicon atoms. In order to run this workflow, we also need the output files from
+calculating the electronic density with CASTEP and a parameter file for our tools.
+These have all been pre-generated for this tutorial, but in general you would
+need to create these yourself outside of Galaxy.
+
+## Get data
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}) or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    ```
+>    https://sandbox.zenodo.org/record/1031546/files/Si-muairss-uep.yaml
+>    https://sandbox.zenodo.org/record/1031546/files/Si-out.cell
+>    https://sandbox.zenodo.org/record/1031546/files/Si.castep
+>    https://sandbox.zenodo.org/record/1031546/files/Si.den_fmt
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets to match their file names, if needed.
+> 4. Check that the datatype is correct for each dataset.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+{: .hands_on}
+
+## Import workflow
+
+> <hands-on-title>Import workflow</hands-on-title>
+>
+> 1. Import the workflow for this tutorial using [these instructions](workflows/).
+> 2. Open the editor for the workflow, and verify that all the required tools
+>    are installed on your Galaxy instance, and have the same version(s) as the
+>    tools in workflow. If this is not the case a dialogue box will warn you.
+>
+{: .hands_on}
+# Finding stopping sites
+
+We can execute the entire workflow in one go, which will generate stopping sites
+based on our four input files. Alternatively, we can run the three tools that
+comprise the workflow one by one. The steps below describe each one in detail:
+what parameters they take and what exactly they do.
+
+## Generate muonated structures (pm-muairss)
```

Comment:
These titles are the old names of the tools. I think we can drop the `(pm-muairss)`
```suggestion
## AIRSS Stage 1: Generate muonated structures
```